app/src/components/main/analytics/events/uniqueErrorsPageEvents.js

```diff
@@ -53,26 +56,23 @@ export const UNIQUE_ERRORS_PAGE_EVENTS = {
   CLICK_ACTIONS_BTN: getClickActionsButtonEvent(UNIQUE_ERRORS_PAGE),
   EDIT_ICON_CLICK: getClickPencilIconEvent(UNIQUE_ERRORS_PAGE),
   getClickOnPlusMinusBreadcrumbEvent: getClickOnPlusMinusEventCreator(UNIQUE_ERRORS_PAGE),
-  // GA3 events
-  ...getCommonActionEvents(UNIQUE_ERRORS_PAGE),
-  onClickIssueTicketEvent: getClickIssueTicketEvent(UNIQUE_ERRORS_PAGE),
-  MAKE_DECISION_MODAL_EVENTS: getMakeDecisionModalEvents(UNIQUE_ERRORS_PAGE),
   CLICK_RUN_BUTTON: {
-    category: UNIQUE_ERRORS_PAGE,
-    action: 'Click on Button "Run Unique Error"',
-    label: '',
+    ...basicUniqueErrorsPageClickEventParameters,
+    element_name: 'run_unique_errors_analysis',
   },
-  clickAnalyzeEvent: getClickAnalyzeInUniqueErrorAnalysisModalEvent(UNIQUE_ERRORS_PAGE),
+  clickAnalyzeEvent: getClickOnAnalyzeUniqueErrorsEventCreator(UNIQUE_ERRORS_PAGE),
   CLICK_CLUSTER_ITEM_ARROW: {
```

Comment:
This event must be triggered only when the user expands the unique error cluster.
Now it works when the user expands and collapses the cluster.

Reply:
Fixed, thank you!